doc/getting_started/installation_linux.rst

```diff
@@ -55,7 +55,7 @@ Install the required packages in a Ubuntu host system with:
 .. code-block:: console
 
    $ sudo apt-get install git make gcc g++ ncurses-dev \
-	 doxygen dfu-util device-tree-compiler
+	 doxygen dfu-util device-tree-compiler python-pip3
```

Comment:
Are you absolutely sure this is called python-pip3 and not python3-pip?
https://packages.ubuntu.com/search?keywords=python3-pip